Makefile

```diff
@@ -15,11 +15,14 @@ lint-jsonnet:
 validate-schemas:
 	./scripts/validate_schemas.sh
 
-translate-schemas:
+eq_translations_check:
```

Comment:
This should be `eq-translations-check` to be consistent with the other target names.

Reply:
Fixed, thanks!